cmd/example-client/main.go

```diff
@@ -133,7 +133,7 @@ func MegaScan(msg info.IDInfo, _ []byte) *connection.SendResponse {
 				Plugins: models.VTsList{
 					Single: []models.SingleVT{
 						{
-							OID: "1.3.6.1.4.1.25623.1.0.90022",
+							OID: "1.3.6.1.4.1.25623.1.0.999900",
```

Comment:
Maybe it is better to use the same OID for the new file so it also works with the normal feed?

Reply:
you mean to set the oid of the test nasl to an existing OID?

Reply:
Yes, just leave this line as it was and change the oid of test.nasl to 1.3.6.1.4.1.25623.1.0.90022